dev/tools/localization/localizations_utils.dart

```diff
@@ -371,7 +371,7 @@ String generateClassDeclaration(
 class $classNamePrefix$camelCaseName extends $superClass {''';
 }
 
-/// Return `s` as a Dart-parseable string.
+/// Return the input string as a Dart-parseable string.
```

Comment:
This looks like we've accidentally made two copies of the same function. Need to include some explanation here.

Reply:
This comment is no longer applicable since the two functions have been consolidated into one